.github/workflows/main.yml

```diff
@@ -60,6 +63,9 @@ jobs:
         run: npm pack
         working-directory: ./npm-publish
 
+      - name: Npm install dependencies for lf-documentation
+        run: npm run install-lf-documentation
+
```

Comment:
can we run the install of the local package a level down?
especially because the "previous" version is already a level down...
I actually think maybe we should specify "DEV_VERSION" in the package.json, and install the local package at the sub level before running npm install on the documentation

Reply:
and then just specify in the README that devs have to install the version they want

Reply:
When you say install it a level down, do you mean like
- node_modules
- projects
  - lf-documentation
  - node_modules